skops/card/_model_card.py

```diff
@@ -214,13 +201,20 @@ def _check_table(self) -> None:
 
     def format(self) -> str:
         if self._is_pandas_df:
-            headers = self.table.columns  # type: ignore
+            pass  # type: ignore
         else:
-            headers = self.table.keys()
+            self.table.keys()
```

Comment:
Do these lines do anything? They can just be removed now, right?

Reply:
Nice catch